src/styles/blocks/card.scss

```
@@ -0,0 +1,79 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  border: $card-border-thickness solid $color-border-grey;
+  border-radius: 5px;
+  background: $color-background-card;
+  transition: all $duration-time $timing-function;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 0 auto 4px;
+    @extend %text-normal;
+    line-height: 18px;
+    color: $color-accent-dark;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    @include flexJustifyAlign(space-between, center);
+    padding: 0;
+  }
+
+  &__review-info {
+    color: $color-accent-dark;
+  }
+
+  &__price {
+    margin-top: 24px;
+    @include flexJustifyAlign(space-between, center);
+  }
+
+  &__price-label {
+    @extend %text-normal;
+    line-height: 18px;
+  }
+  
+  &__price-value {
+    @extend %text-large;
+    line-height: 18px;
+    color: $color-accent-dark;
+  }
```

one more level of nesting could be added
```suggestion
  &__price {
    margin-top: 24px;
    @include flexJustifyAlign(space-between, center);

    &-label {
      @extend %text-normal;
      line-height: 18px;
    }
  
    &-value {
      @extend %text-large;
      line-height: 18px;
      color: $color-accent-dark;
    }
  }
```